src/cmd/mod.rs

```diff
@@ -92,15 +96,47 @@ pub async fn commit(all: bool, amend: bool, term: &Term) -> Result<()> {
         return Ok(());
     }
 
-    let CommitTitleDescription { title, description } =
-        ask_commit_title_description(&config, term).await?;
+    let commit = match read_recovery_file() {
+        Ok(Some(commit)) => match ask_for_recovery(term, commit.clone()) {
```

Comment:
💭 **thought**: Maybe we could improve the UX if we use the recovery value as default value for title & description.

It's nearly as fast for the user, and allow update value if necessary